src/kata7.getEmployerRole.js

```diff
@@ -1,3 +1,6 @@
-const getEmployerRole = (employeeName, employees) => {};
+const getEmployerRole = (employeeName, employees) => {
+  // Seeing as there is only one of each name...
```

Comment:
It's good you are thinking this way. If you were to expand the tests here so that it would return all employees with a given name if there were multiple values, how would you refactor it?

Reply:
I think I'd probably update my test to be looking for an array of each employee object who's name matches the employeeName argument and then I'd update my function to use filter instead of find `return employees.filter(employee => employee.name === employeeName);`, would that be right? But I think the easier solution would be to add some unique identifier to the employee and search on that rather than name, like an employee number maybe 😂

Reply:
Great answer! And you're right, you want a unique identifier for your data where possible. Lots of people can have the same name, so a unique ID for every new person would be ideal.